### ISSUE TITLE
great work

### ISSUE THREAD

**Issue by hyq1860** [DavidFuchs42/TALibraryInCSharp#1]
thank you!

**Comment by DavidFuchs42** [comment]
Thank you for noticing this little library ;-) 

The conversion was almost fully automated, except for the 5000 errors I needed to write code to fix, and get this thing into safe mode again. 

The next version is TAMagic, it will contain the routines for 
- Financial Data (High Low Open Close Datetime volume)
- the TA routines
- grouped TA (ie multiple TA routines combined for entry and exit of trades)
- back testing
- Database Access for all routines

**Comment by rossspaul** [comment]
Thank you so much for creating this library for C# developers.  Please keep it going, very much appreciated!!!

**Comment by DavidFuchs42** [comment]
I am still working on this, added some wrappers and some routine to make it easier to save and load optional function settings. Plus all the databasing needed. I will upload it when I am done, it is kind of a kludge right now.

**Comment by mamohanc** [comment]
> The conversion was almost fully automated, except for the 5000 errors I needed to write code to fix, and get this thing into safe mode again.

Yeah just like i walked out of my mother's womb, well except for the nine months of her carrying me of course. 

Great work dude.

**Comment by DavidFuchs42** [comment]
> Yeah just like i walked out of my mother's womb, well except for the nine months of her carrying me of course.
> 
> Great work dude.

Thanks
